app/formatters.py

```diff
@@ -98,6 +98,13 @@ def format_datetime_short(date):
     )
 
 
+def format_datetime_short_12h(date):
+    # example: February 20, 2024 at 07:00 PM US/Eastern
+    return "{} at {} {}".format(
```

Comment:
I think I'm getting a little confused about how many datetime formats are needed for our app.  I know @maomeara63 wrote a ticket that caused me to change all the 'format_datetime_short' formats to 'format_datetime_short_america', but now 'format_datetime_short' is coming back in a new form.  Do we really want dates in different formats in different places?  I would prefer UTC 24 hour format everywhere but I know I'm the only one who wants that. :)

Reply:
@terrazoon Good point.

So far, these are the two date formats that have been connected with the redesign. I would be okay if we standardized the dates to fit these formats as well or just one format 

03-01-2024 at 01:48 PM
February 20, 2024 at 07:00 PM US/Eastern